Sources/TSCUtility/StringExtensions.swift

```diff
@@ -96,9 +96,12 @@ extension String {
     }
 
     public func spm_multilineIndent(count: Int) -> String {
-        return self
-            .split(separator: "\n")
-            .map{ String(repeating: " ", count: count) + $0 }
-            .joined(separator: "\n")
+        let indent = String(repeating: " ", count: count)
+        let fullRange = self.startIndex ..< self.endIndex
+        var ret = ""
+        self.enumerateSubstrings(in: fullRange, options: [.byLines, .substringNotRequired]) { _, _, enclosingRange, _ in
+            ret += indent + self[enclosingRange]
+        }
+        return ret
```

Comment:
Thanks for looking at this! Have you done any performance comparison with this change? I wonder if bridging to ObjC hurts performance on macOS. 

cc  @milseman

Reply:
Actually you were right! Implicit bridging made fix ~25 times slower than original implementation on file of 2000 lines. So I rolled back original implementation and just applied slight optimisation with moving string creation out of closure (~9% gain on a 2000 lines file).

Reply:
Great, thank you!